Police-Data-Accessibility-Project/design-system
# feat(components): loading

- Adds wrapper around tailwind utility for loading shimmer
- Adds loading spinner

Will take care of #63 when merged to main


## To test
1. check out this branch, `npm run dev`
2. On the demo page, check out the loading class and spinner
